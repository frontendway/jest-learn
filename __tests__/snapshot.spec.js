import { generatorConfig } from '../src/snapshot/index.js'

it('生成 config 配置', () => {
  expect(generatorConfig()).toMatchSnapshot()
})
