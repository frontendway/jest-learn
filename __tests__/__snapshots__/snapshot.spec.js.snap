// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`生成 config 配置 1`] = `
Object {
  "port": 8080,
}
`;
